crates/gateway-client/src/builder.rs

```diff
@@ -364,9 +352,9 @@ where
     use pathfinder_retry::Retry;
     use std::num::NonZeroU64;
 
-    Retry::exponential(future_factory, NonZeroU64::new(2).unwrap())
-        .factor(NonZeroU64::new(15).unwrap())
-        .max_delay(std::time::Duration::from_secs(10 * 60))
+    Retry::exponential(future_factory, NonZeroU64::new(1).unwrap())
+        .factor(NonZeroU64::new(2).unwrap())
+        .max_delay(std::time::Duration::from_secs(30))
```

Comment:
I think we definitely want this change in at least.

Reply:
Was:
`min(2^N x 15, 600) [secs]`
Is:
`min(1^N x 2, 30) [secs]`
I guess you meant:
`min(2^N, 30) [secs]`

```suggestion
    Retry::exponential(future_factory, NonZeroU64::new(2).unwrap())
        .max_delay(std::time::Duration::from_secs(30))
```